### PR TITLE
Proofread and Improve security-basics.rst

### PR DESCRIPTION
**Pull request by NPDebs**
Contribution for outreachy [#41](https://github.com/jupyterhub/outreachy/issues/41).

For ease in assimilation, I read and made these changes to the documentation:

- Corrected typos.
- Improved sentence structure.
- Used initial caps where needed.

Please review. Thank you!

jupyterhub-pr

### CODE REVIEW

File: docs/source/getting-started/security-basics.rst [hunk 1]
```diff
@@ -1,12 +1,12 @@
-Security settings
```
Comment (minrk) [inline docs/source/getting-started/security-basics.rst:1]: Please keep section headings in "Sentence case" not "Title Case" (here and everywhere below)

File: docs/source/getting-started/security-basics.rst [hunk 2]
```diff
@@ -44,7 +44,7 @@ Some cert files also contain the key, in which case only the cert is needed. It
 is important that these files be put in a secure location on your server, where
 they are not readable by regular users.
 
-If you are using a **chain certificate**, see also chained certificate for SSL
+If you are using a **chain certificate**, see also, chained certificate for SSL
```
Comment (minrk) [inline docs/source/getting-started/security-basics.rst:47]: The added comma is not correct.

File: docs/source/getting-started/security-basics.rst [hunk 3]
```diff
@@ -115,29 +115,29 @@ using the ``CONFIGPROXY_AUTH_TOKEN`` environment variable:
 
 This environment variable needs to be visible to the Hub and Proxy.
 
-Default if token is not set
+Default if Token is Not Set
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If you don't set the Proxy authentication token, the Hub will generate a random
-key itself, which means that any time you restart the Hub you **must also
+key itself. This means that anytime you restart the Hub, you **must also
```
Comment (minrk) [inline docs/source/getting-started/security-basics.rst:122]: two words `any time` was correct before

File: docs/source/getting-started/security-basics.rst [hunk 4]
```diff
@@ -115,29 +115,29 @@ using the ``CONFIGPROXY_AUTH_TOKEN`` environment variable:
...
 generating and configuring the cookie secret.
 
-Generating and storing as a cookie secret file
+Generating and Storing as a Cookie Secret File
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The cookie secret should be 32 random bytes, encoded as hex, and is typically
-stored in a ``jupyterhub_cookie_secret`` file. An example command to generate the
-``jupyterhub_cookie_secret`` file is:
+stored in a ``jupyterhub_cookie_secret`` file. Below, is an example of a command to generate the
```
Comment (minrk) [inline docs/source/getting-started/security-basics.rst:139]: No comma here. Let's also drop 'of a', 'example command' is still fine.

File: docs/source/getting-started/security-basics.rst [hunk 5]
```diff
@@ -155,10 +155,10 @@ The location of the ``jupyterhub_cookie_secret`` file can be specified in the
 
 If the cookie secret file doesn't exist when the Hub starts, a new cookie
 secret is generated and stored in the file. The file must not be readable by
-``group`` or ``other`` or the server won't start. The recommended permissions
+``group`` or ``other``, otherwise, the server won't start. The recommended permissions
```
Comment (minrk) [inline docs/source/getting-started/security-basics.rst:158]: Second comma is not correct

File: docs/source/getting-started/security-basics.rst [hunk 6]
```diff
@@ -235,7 +234,7 @@ shared by the Hub and single-user servers.
 
 Its sole purpose is to coordinate logout of the multiple OAuth cookies.
 
-This cookie is set to ``/`` so all endpoints can receive it, or clear it, etc.
+This cookie is set to ``/`` so all endpoints can receive or clear it, etc.
```
Comment (minrk) [inline docs/source/getting-started/security-basics.rst:237]: Let's go with the parallel construction:

```suggestion
This cookie is set to ``/`` so all endpoints can receive it, clear it, etc.
```

**Review by minrk — COMMENTED** [review]
Thanks for the PR! I've left some comments inline for you to address. It may be easiest to revert all of the heading changes.

A good tip for contributing to projects in general: If a systematic change (e.g. Title Case Headings) would change every instance in a project, then it is a good assumption that the project uses a different system (e.g. "Sentence case headings"), rather than having made errors in every case. If you're not sure, it's best to ask first if it looks like a huge amount of changes are likely to result from a systematic change like this.

**Comment by NPDebs** [comment]
Hi @minrk. 👋
Judging by the number of ways in which I messed up this documentation, it's extremely generous of you to thank me for the PR.

English is my official language. Yet, this is the first time I'm made to understand the difference between _Title Case Headings_ and _Sentence case headings_. This is the beauty of open-source, I think. The opportunity to draw from an infinite sea of knowledge and experience.

Thank you for taking the time to scrutinise this PR. I have made the necessary changes, as directed...and I now know to ask before making dramatic changes. 😃

Have a lovely weekend. 🙌

**Comment by NPDebs** [comment]
Hi @minrk. 👋
Judging by the number of ways in which I messed up this documentation, it's extremely generous of you to thank me for the PR.

English is my official language. Yet, this is the first time I'm made to understand the difference between _Title Case Headings_ and _Sentence case headings_. This is the beauty of open-source, I think. The opportunity to draw from an infinite sea of knowledge and experience.

Thank you for taking the time to scrutinise this PR. I have made the necessary changes, as directed...and I now know to ask before making dramatic changes. 😃

Have a lovely weekend. 🙌

File: docs/source/getting-started/security-basics.rst (hunk 1 above)
Comment (NPDebs) [reply docs/source/getting-started/security-basics.rst]: Done. Thank you...

File: docs/source/getting-started/security-basics.rst (hunk 4 above)
Comment (NPDebs) [reply docs/source/getting-started/security-basics.rst]: Thank you.

**Comment by minrk** [comment]
Great updates, thank you!

**Pull request merged**